internal/player/player_test.go

```diff
@@ -0,0 +1,29 @@
+package player
+
+import "testing"
+
+func TestCreatePlayer(t *testing.T) {
+	tables := []struct {
```

Comment:
The idea would be to use the player struct. You can use it directly as we are in the same package. If not, then when we modify the player struct, we need to come and change this one too.
tables := Player{
...
}

Reply:
Agreed, will do

Reply:
Done, I think. Unrelated thing for later: If we have a public constructor for Player (and create some public accessors), maybe make Player type private?